src/runtime_src/core/edge/user/aie/aie.cpp

```diff
@@ -338,32 +342,50 @@ start_profiling(int option, const std::string& port1Name, const std::string& por
   auto gmio = std::find_if(gmios.begin(), gmios.end(),
             [port1Name](gmio_type it) { return it.name.compare(port1Name) == 0; });
 
-  if (gmio == gmios.end())
+  // For PLIO inside graph, there is no name property.
+  // So we need to match logical name too
+  auto plio = std::find_if(plios.begin(), plios.end(),
+            [port1Name](plio_type it) { return it.name.compare(port1Name) == 0; });
```

Comment:
`[&port1Name] ` prevents a string copy
and use const plio_type&, e.g. this way:
`[&port1Name] (auto& pt) { return pt.name.compare(port1Name) == 0; }`
Personally I don't like camel case so I'd prefer port1_name but that's just me.

Reply:
done